kmcd-props.md

```diff
@@ -126,13 +126,13 @@ Art of an ilk = Sum of all urn art across all users for that ilk.
 ```k
     syntax Wad ::= sumOfUrnArt(Map, String, Wad) [function, functional]
  // -------------------------------------------------------------------
-    rule sumOfUrnArt( {ILKID , ADDR} |-> Urn (... art: ART) URNS, ILKID', SUM)
-      => #if ILKID ==K ILKID'
-            #then sumOfUrnArt( URNS, ILKID', SUM +Wad ART)
-            #else sumOfUrnArt( URNS, ILKID', SUM)
+    rule sumOfUrnArt( {ILK_ID , _ADDR} |-> Urn (... art: ART) URNS, ILK_ID', SUM)
```

Comment:
Could we just use `_` instead of `_ADDR`?

Reply:
Done.